oap-server/server-starter/src/main/resources/ui-initialized-templates/elasticsearch/elasticsearch-cluster.json

```diff
@@ -0,0 +1,454 @@
+                  },
+                  "widget":{
+                    "title":"Unassigned shards"
+                  }
+                },
+                {
+                  "x":3,
+                  "y":0,
+                  "w":3,
+                  "h":6,
+                  "i":"12",
+                  "type":"Widget",
+                  "id":"0-0-12",
+                  "metricTypes":[
+                    "readMetricsValues"
+                  ],
+                  "metrics":[
+                    "meter_elasticsearch_cluster_breakers_tripped"
+                  ],
+                  "moved":false,
+                  "graph":{
+                    "type":"Card",
+                    "fontSize":14,
+                    "textAlign":"center",
+                    "showUnit":true
+                  },
+                  "widget":{
+                    "title":"Tripped of breakers"
+                  }
+                },
+                {
+                  "x":6,
+                  "y":0,
+                  "w":3,
+                  "h":6,
+                  "i":"13",
+                  "type":"Widget",
+                  "id":"0-0-13",
+                  "metricTypes":[
+                    "readMetricsValue"
+                  ],
+                  "metrics":[
+                    "meter_elasticsearch_cluster_cpu_usage_avg"
+                  ],
+                  "moved":false,
+                  "graph":{
+                    "type":"Card",
+                    "fontSize":14,
+                    "textAlign":"center",
+                    "showUnit":true
+                  },
+                  "widget":{
+                    "title":"CPU usage Avg (%)"
+                  }
+                },
+                {
+                  "x":9,
+                  "y":0,
+                  "w":3,
+                  "h":6,
+                  "i":"14",
+                  "type":"Widget",
+                  "id":"0-0-14",
+                  "metricTypes":[
+                    "readMetricsValue"
+                  ],
+                  "metrics":[
+                    "meter_elasticsearch_cluster_jvm_memory_used_avg"
+                  ],
+                  "moved":false,
+                  "graph":{
+                    "type":"Card",
+                    "fontSize":14,
+                    "textAlign":"center",
+                    "showUnit":true
+                  },
+                  "widget":{
+                    "title":"JVM memory used Avg (%)"
+                  }
+                },
+                {
+                  "x":21,
+                  "y":0,
+                  "w":3,
+                  "h":6,
+                  "i":"15",
+                  "type":"Widget",
+                  "id":"0-0-15",
+                  "metricTypes":[
+                    "readMetricsValue"
+                  ],
+                  "metrics":[
+                    "meter_elasticsearch_cluster_open_file_descriptors"
```

Comment:
Maybe is a typo too?  It should be `meter_elasticsearch_cluster_open_file_count`?

Reply:
It seems that I did not save the dashboard after modifying it through the UI.

Reply:
> 1. You should keep upper/lower cases consistent. Such as put all first letters of words in upper case.
> 2. `/s` should be `(/s)` or `Per Second`
> 
> After the e2e tests added and merged, you should update showcase to add ElasticSearch monitoring too.

Sure, I will write a showcase.